float64[7] tau_J
float64[7] tau_J_d
float64[7] tau_d
float64[7] tau_task
float64[7] gravity
float64[7] coriolis
float64[7] tau_nullspace

